commands/screen.js

```diff
@@ -1,6 +1,6 @@
 // @cliDescription  Generates an opinionated container.
 
-module.exports = async function (context) {
+module.exports = async function(context) {
```

Comment:
This needs a space between function and (context).  You can run `npm run standard` to find these issues.